platform/darwin/src/MGLRasterSource.h

```diff
@@ -0,0 +1,10 @@
+#import <Mapbox/Mapbox.h>
```

Comment:
Looks like removing Mapbox.h also requires either UIKit or CoreGraphics to be added.

```
#import "MGLSource.h"
#import <CoreGraphics/CoreGraphics.h>
```